src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -20,6 +21,8 @@ public PullRequestCreationView()
 
             this.WhenActivated(d =>
             {
+                d(this.BindCommand(ViewModel, vm => vm.Cancel, v => v.cancelButton));
```

Comment:
We don't need to do this here, just add a `Command="{Binding Cancel}"` in the xaml and it'll automagically work.